src/renderer/components/KeyboardShortcutsDialog.tsx

```diff
@@ -0,0 +1,205 @@
+import {
+  Dialog,
+  Box,
+  Typography,
+  Stack,
+  styled,
+  List,
+  ListItem,
+} from '@mui/material';
+import { useEffect, useState } from 'react';
+import colors from 'renderer/colors';
+import { PrimaryButton } from 'renderer/components/Blocks/Buttons';
+import ipc from 'renderer/ipc';
+import { OperatingSystems } from 'sharedTypes';
+
+const CustomStack = styled(Stack)`
+  width: 100%;
+  height: 100%;
+`;
+
+const CustomModal = styled(Dialog)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const CustomModalInner = styled(Box)`
+  background: ${colors.grey[700]};
+  padding: 15px 30px 30px 30px;
+  border-radius: 5px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  width: 544px;
+  height: 550px;
+`;
+
+const CustomColumnStack = styled(CustomStack)`
+  flex-direction: row;
+  align-items: start;
+  justify-content: space-between;
+`;
+const CustomRowStack = styled(CustomStack)`
+  padding-top: 80px;
+`;
+
+const CustomListItem = styled(ListItem)`
+  padding: 0 0 0 0;
+`;
+
+interface RowProps {
+  wordsLeft: string[];
+  shortcutsLeft: string[];
+  wordsRight: string[];
+  shortcutsRight: string[];
+}
+
+interface Props {
+  open: boolean;
+  onClose: () => void;
+}
+
+const CustomRow = ({
+  wordsLeft,
+  shortcutsLeft,
+  wordsRight,
+  shortcutsRight,
+}: RowProps) => {
+  return (
+    <CustomColumnStack>
+      <List>
+        {wordsLeft.map((word: string) => (
+          <CustomListItem>
+            <Typography
+              variant="p-400"
+              paddingTop="0px"
+              color={colors.grey[300]}
+            >
+              {word}
+            </Typography>
+          </CustomListItem>
+        ))}
+      </List>
+      <List>
+        {shortcutsLeft.map((shortcut: string) => (
+          <CustomListItem>
+            <Typography
+              variant="p-400"
+              paddingTop="0px"
+              color={colors.yellow[500]}
+              width="125px"
+            >
+              {shortcut}
+            </Typography>
+          </CustomListItem>
+        ))}
+      </List>
+      <List>
+        {wordsRight.map((word: string) => (
+          <CustomListItem>
+            <Typography
+              variant="p-400"
+              paddingTop="0px"
+              color={colors.grey[300]}
+            >
+              {word}
+            </Typography>
+          </CustomListItem>
+        ))}
+      </List>
+      <List>
+        {shortcutsRight.map((shortcut: string) => (
+          <CustomListItem>
+            <Typography
+              variant="p-400"
+              paddingTop="0px"
+              color={colors.yellow[500]}
+              width="125px"
+            >
+              {shortcut}
+            </Typography>
+          </CustomListItem>
+        ))}
+      </List>
+    </CustomColumnStack>
+  );
+};
+
+const KeyboardShortcutsDialog = ({ open, onClose }: Props) => {
+  const [currentOs, setCurrentOs] = useState<OperatingSystems | null>(null);
```

Comment:
OS is not something that ever changes, so I’d almost be tempted to say make this a global var that loads outside the react context - this is fine for now though